service/src/test/java/bio/terra/drshub/services/DrsProviderServiceTest.java

```diff
@@ -56,20 +56,33 @@ void testHostnameIdResolution() {
     }
   }
 
+  @Test
+  void testHostnameMatchingIsCaseInsensitive() {
+    var expectedUrl =
+        "drs://jade.datarepo-dev.broadinstitute.org/v1_e2151834-13cd-4156-9ea2-168a1b7abf60_0761203d-d2a1-448e-8f71-9f81d80ddd9d";
```

Comment:
Is the expectation that tests are only run in the dev tier?

If tests are also run in other tiers, the tier in the hostname should be parameterized.

Reply:
Thanks for your support @Shakespeared!
I am happy with this as it stands.